.github/ISSUE_TEMPLATE/question.yml

```
@@ -0,0 +1,133 @@
+name: ❓ Question
+description: Ask a question about the project
+title: "❓ [Question]: "
+labels: ["question", "triage"]
+assignees: []
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Have a question about Viscalyx.se? We're here to help!
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)